contrib/world_builder/include/world_builder/bounding_box.h

```diff
@@ -93,7 +93,7 @@ namespace WorldBuilder
    *
    * @note The majority of this class is copied from the deal.II library.
    */
-  template <int spacedim>
+  template <unsigned int spacedim>
```

Comment:
Wait, you really did make that change? Maybe I need to reconsider #5844 :-)

Reply:
It is the easiest way to get rid of the new warnings when using clang/the new gcc flag I added in he world builder and logically the most consistent. I don't think it will impact ASPECT in any way, since the types are not shared. For aspect it may be more difficult since a lot is shared with dealii, but maybe not.